ui/DesignSystem/lib/unreachable.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+export const unreachable = (_: never): void => {};
```

Comment:
We should probably throw an error and attach the argument so that we know at runtime if something with the types is off. The return type should also be `never` so that we can use this in any place where we need to return something. For example https://github.com/radicle-dev/radicle-upstream/blob/4be4f1a9a850e49ddfbe2eab7933e067183366fd/ui/src/network.ts#L70

Reply:
b5740ea07de0f789823edcaae1ee976ee1a588e6